suggestion for privacy FAQ (#40)
did-app/www: faq/index.md

Password managers are a way to manage and remember large numbers of passwords but they fundamentally don’t solve the security issues with password theft.  A user can still be tricked into handing over their password during a phishing attack. The password manager's ‘master password’ grants access to all username and password combinations stored in the password manager and is vulnerable to the same risks as any other password.  Password managers are a high-value target for hackers.

Using a password manager is a choice the user has to make and, in some cases, pay for.  The website owner has no control over how well the user looks after their passwords either in a password manager or not.  DID.app is different because the website owner chooses to use it to improve user experience and remove the password problem.

### How does DID.app handle privacy?

> Authenticating users requires the use of an email address.  Email addresses are considered personally identifiable information.  When an end-user signs in with DID.app, the service collects and stores the user's email address.

Reviewer: ```suggestion
Authenticating users requires them to submit an email address. 
Email addresses are considered personally identifiable information. 
DID.app stores user email address, to provide it's secure authentication service.
```

Putting each sentence on a new line. Also I would avoid "The service" because it could be DID or the third party

> This email address can be shared with the website running DID.app if you need it to be for the purpose of communicating with the user, however, the email address is not needed to identify the user.  This is achieved with the Client ID which is not personally identifiable information.

Reviewer: ```suggestion
User email addresses can be accessed by your website when using DID.app.
However, the email address is not needed to identify the user.
User identities are tied to the `sub` claim in the signed ID Token, the sub value is different for each site a user visits
```

Tried to use less "This" . I find it weird when a new paragraph starts with this.

> When running DID.app on your website you do not have to collect or store any PII (personally identifiable information) from the authentication service in order to authenticate users.

Reviewer: ```suggestion
When using DID.app on your website you do not have to collect or store any Personally Identifiable Information (PII) to authenticate users.
```

"PII from the authentication service" or "PII to authenticate users" but  both seems verbose to me.
